staging/src/k8s.io/api/storage/v1beta1/types.go

```diff
@@ -66,6 +66,12 @@ type StorageClass struct {
 	// the VolumeScheduling feature.
 	// +optional
 	VolumeBindingMode *VolumeBindingMode `json:"volumeBindingMode,omitempty" protobuf:"bytes,7,opt,name=volumeBindingMode"`
+
+	// Snapshotter is the driver expected to handle this StorageClass.
+	// This is an optionally-prefixed name, like a label key.
+	// For example: "kubernetes.io/gce-pd" or "kubernetes.io/aws-ebs".
+	// This value may not be empty.
+	Snapshotter string `json:"snapshotter,omitempty" protobuf:"bytes,8,opt,name=snapshotter"`
```

Comment:
Why is this also added in v1beta1?  We should start with v1alpha1 and move to v1beta1 later.